k8s/templates/job-template.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "studio.fullname" . }}-django-config
+  labels:
+    app: {{ template "studio.fullname" . }}
+data:
+  DJANGO_SETTINGS_MODULE: {{ . Values.settings }}
+  DJANGO_LOG_FILE: /var/log/django.log
+  MPLBACKEND: PS
+  STUDIO_BETA_MODE: "yes"
+  RUN_MODE: k8s
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "studio.fullname" . }}-db-creds
+  labels:
+    app: {{ template "studio.fullname" . }}
+type: Opaque
+data:
+  DATA_DB_USER: {{ .Values.postgresql.postgresUser | default "" | b64enc }}
+  DATA_DB_PASS: {{ .Values.postgresql.postgresPassword | default "" | b64enc }}
+  DATA_DB_HOST: {{ .Values.postgresql.externalCloudSQL.proxyHostName | default (include "postgresql.fullname" .) }}
+  DATA_DB_NAME: {{ .Values.postgresql.postgresPassword | default "" | b64enc }}
+---
+apiVersion: apps/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "studio.fullname" . }}-job-template
+  labels:
+    app: {{ template "studio.fullname" . }}
+  annotations:
+    description: "This Cron Job isn't actually supposed to run in daily operations. Instead it's used as a template for migration jobs, that we can copy using `kubectl create job --from=<cronjob>`."
+spec:
+  schedule: "@yearly"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: OnFailure
+          containers:
+          - name: dbmigrate
+            image: {{ .Values.studioApp.imageName }}
+            args:
+            - make
+            - migrate
+            envFrom:
+            - configMapRef:
+                name: {{ template "studio.fullname" . }}-db-config
```

Comment:
where is this configMap `{{ template "studio.fullname" . }}-db-config` defined? I don't seem to find it

Reply:
Good catch! I removed this now.